doc/radosgw/notifications.rst

```diff
@@ -394,31 +394,33 @@ The response has the following format:
         </ResponseMetadata>
     </GetTopicAttributesResponse>
 
-- User: the name of the user that created the topic.
-- Name: the name of the topic.
-- EndPoint: The JSON-formatted endpoint parameters, including:
-   - EndpointAddress: The push-endpoint URL.
-   - EndpointArgs: The push-endpoint args.
-   - EndpointTopic: The topic name to be sent to the endpoint (can be different
-     than the above topic name).
-   - HasStoredSecret: This is "true" if the endpoint URL contains user/password
+- ``User``: the name of the user that created the topic
+- ``Name``: the name of the topic
+- ``EndPoint``: the JSON-formatted endpoint parameters, including:
+
+   - ``EndpointAddress``: the push-endpoint URL
+   - ``EndpointArgs``: the push-endpoint arguments
+   - ``EndpointTopic``: the topic name to be sent to the endpoint (can be different
```

Comment:
Would be nice to capitalize `The` here to be consistent.

Reply:
I suggest that we consistently capitalize list items.  They only get a period at the end if they're sentences.  These things are minor, we can always follow up on our own PR that doesn't block devs.